cmd/get/matchable_workflow_execution_config.go

```diff
@@ -97,3 +105,30 @@ func getWorkflowExecutionConfigFunc(ctx context.Context, args []string, cmdCtx c
 	}
 	return nil
 }
+
+
+func getSampleWorkflowExecutionFileConfig(project, domain, workflow string) workflowexecutionconfig.FileConfig {
+	return workflowexecutionconfig.FileConfig{
+		Project: project,
+		Domain:  domain,
+		Workflow: workflow,
+		WorkflowExecutionConfig: &admin.WorkflowExecutionConfig{
```

Comment:
this is great, but it's too bad we have to manually update this if the proto structure changes. is there anyways to use reflection somehow to populate all the fields from the proto definition?

Reply:
So we can dump it in following way if we enable EmitDefaults in jsnonpb.Marshaller.
The user still has to know the types for each of the fields when modifying it.

```
annotations: null
labels: null
maxParallelism: 0
rawOutputDataConfig: null
securityContext: null
```
Having a customized default lets us give a sample structure with the populated types which is something like this.
And also this gives us to go beyond one level and define deeper customized objects instead of default which would be nil.

```
annotations:
  values:
    cliAnnotationKey: cliAnnotationValue
domain: development
labels:
  values:
    cliLabelKey: cliLabelValue
max_parallelism: 10
project: flytesnacks
raw_output_data_config:
  output_location_prefix: cliOutputLocationPrefix
security_context:
  run_as:
    k8s_service_account: default
```

I understand its going to be maintenance to add a new field and requires a corresponding change in the default but doesn't  this provide a more user friendly option. 
Let me know what you think.

Reply:
prioritizing user-friendliness definitely is the superior option! I don't imagine we'll update the proto structure too frequently so this seems like a superior choice 👍